documentation/content/guidebook/drizzle-orm/index.md

```diff
@@ -255,44 +255,38 @@ Make sure to change the table names accordingly.
 
 ```ts
 // schema.js
-import { pgTable, bigint, varchar, boolean } from "drizzle-orm/pg-core";
+import { sqliteTable, text, blob } from "drizzle-orm/sqlite-core";
 
-export const user = pgTable("user", {
-	id: varchar("id", {
-		length: 15 // change this when using custom user ids
-	}).primaryKey()
-	// other user attributes
+export const user = sqliteTable("user", {
+  id: text("id").primaryKey(),
+  // other user attributes
 });
 
-export const session = pgTable("user_session", {
-	id: varchar("id", {
-		length: 128
-	}).primaryKey(),
-	userId: varchar("user_id", {
-		length: 15
-	})
-		.notNull()
-		.references(() => user.id),
-	activeExpires: bigint("active_expires", {
-		mode: "number"
-	}).notNull(),
-	idleExpires: bigint("idle_expires", {
-		mode: "number"
-	}).notNull()
+export const session = sqliteTable("user_session", {
+  id: text("id").primaryKey(),
+  userId: text("user_id", {
+    length: 15,
```

Comment:
missed this

Reply:
updated at [Commit](https://github.com/pilcrowOnPaper/lucia/pull/934/commits/bf0401f069958dbb8ffd67cde8c2900d869132b9)